### PR TITLE
<WIP> Add support for multiple Kafka Clusters to Control Center

### PR DESCRIPTION
**Pull request by nerdynick**
# Description

Add similar support as multiple connect/ksql to supporting multiple Kafka Clusters.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Currently working with customer to verify, test, and debug.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by JumaX — CHANGES_REQUESTED** [review]
@nerdynick We would recommend rebasing this to merge into the 6.0.x branch since it's a new feature.  Additionally, we have completely re-written how properties and configs are applied to a host for the 6.0 release.

For example, you can see the updated template here: https://github.com/confluentinc/cp-ansible/blob/6.0.x/roles/confluent.control_center/templates/control-center.properties.j2

All of the variables are now pulled fro the confluent.variables role and merged into a final list of properties which are then pulled by the control-center.properties.j2.  

For example:

https://github.com/confluentinc/cp-ansible/blob/6.0.x/roles/confluent.variables/defaults/main.yml

To summarize, I would recommend pointing this towards the 6.0.x branch and refactoring to use the new variable merging structure.

**Comment by nerdynick** [comment]
This is rebased and addressed as part of PR #577
